src/EventListener/TranslatingApiProblemListener.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\EventListener;
+
+use FluentDOM\DOM\Element;
+use Libero\ApiProblemBundle\Event\CreateApiProblem;
+use Symfony\Component\Translation\TranslatorInterface;
+
+trait TranslatingApiProblemListener
+{
+    use SimplifiedApiProblemListener;
+
+    final protected function createTitle(CreateApiProblem $event) : string
+    {
+        return $this->translate($this->getTitle($event), $event);
+    }
+
+    final protected function createDetails(CreateApiProblem $event) : ?string
+    {
+        if (!$translation = $this->getDetails($event)) {
+            return null;
+        }
+
+        return $this->translate($translation, $event);
+    }
+
+    abstract protected function getTitle(CreateApiProblem $event) : Translation;
```

Comment:
I wans't aware of the connections between the single fields that make up a `Translation` (key and parameters), but with this model in mind the next level of abstraction is to unify `getTitle()` and `getDetails()` into returning a `Translation[]`. `supports`/`createStatus` don't really fit with those instead.

Reply:
Could postpone for now? Not sure another level is needed, but we can always introduce one if it becomes a problem.

Reply:
It's a breaking change to this interface, but we are in `0.x` so it's always allowed for cleaning up